### PR TITLE
Enable build on win64.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ build:
   rpaths:
     - lib/R/lib/
     - lib/
-  skip: True  # [win]
+  skip: True  # [win32]
```
Comment (jakirkham) [inline recipe/meta.yaml:22]: When making this sort of change, one need to [re-render]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) the feedstock.

**Comment by cbrueffer** [comment]
Done, thanks!

**Comment by jakirkham** [comment]
np. In future PRs, can just [ping our bot]( https://conda-forge.org/docs/webservice.html#conda-forge-admin-please-rerender ) to do the re-rendering for you. Since this one was already merged, we didn't have that option.

**Comment by cbrueffer** [comment]
Not sure I understand; do I do this in the PR I'm about to merge, but before merging?

**Comment by jakirkham** [comment]
Right. This is something that you would do in an open PR.

**Comment by cbrueffer** [comment]
I'll keep it in mind, thanks!

**Pull request merged**